app/views/user_mailer/claim_notification.text.erb

```diff
@@ -5,7 +5,7 @@
 
   You can read announcements about recent archive moves at AO3 News (<%= root_url %>admin_posts?tag=18), and find additional information on Open Doors's FAQ page (http://opendoors.transformativeworks.org/faq) or tutorials page (http://opendoors.transformativeworks.org/tutorials). For any questions not answered in the FAQ, tutorials, or this e-mail, please contact AO3 Support at <%= root_url %>support.
 
-  If this is a mistake and these are not your works, please don't delete them! Please contact Open Doors (http://transformativeworks.org/contact/open%20doors) and we will sort it out.
+  If this is a mistake and these are not your works, please don't delete them! Please contact Open Doors ( https://opendoors.transformativeworks.org/en/contact-open-doors/ ) and we will sort it out.
```

Comment:
Could you remove the spaces that were added inside the parentheses?

Reply:
Sure -- I added them in case some email programs included the close paren in the URL, but I'll remove them.